src/themes.rs

```diff
@@ -0,0 +1,275 @@
+use serde::Deserialize;
+use structopt::clap::{_clap_count_exprs, arg_enum};
+
+arg_enum! {
+    #[derive(Debug, Deserialize, Clone)]
+    #[serde(rename_all = "lowercase")]
+    pub enum ColorScheme {
+        Archlinux,
+        Zenburn,
+        Monokai,
+        Squirrel,
+    }
+}
+
+impl ColorScheme {
+    /// Returns the URL-compatible name of a color scheme
+    pub fn to_string(&self) -> String {
+        match &self {
+            ColorScheme::Archlinux => "archlinux",
+            ColorScheme::Zenburn => "zenburn",
+            ColorScheme::Monokai => "monokai",
+            ColorScheme::Squirrel => "squirrel",
+        }
+        .to_string()
+    }
+
+    /// Returns wether a color scheme is dark
+    pub fn is_dark(&self) -> bool {
+        match &self {
+            ColorScheme::Archlinux => true,
+            ColorScheme::Zenburn => true,
+            ColorScheme::Monokai => true,
+            ColorScheme::Squirrel => false,
+        }
+    }
+
+    /// Returns the name of a color scheme
+    pub fn get_name(&self) -> String {
```

Comment:
And here, maybe use https://github.com/Peternator7/strum#intostaticstr?

Reply:
I'll check this crate :)